apps/docs/src/routes/extras/gltf.md

```diff
@@ -73,7 +73,8 @@ frustumCulled: boolean | undefined = undefined
 renderOrder: number | undefined = undefined
 visible: boolean | undefined = undefined
 dispose: boolean | undefined = undefined
-dracoDecoderPath: string | undefined = undefined
+d̶r̶a̶c̶o̶D̶e̶c̶o̶d̶e̶r̶P̶a̶t̶h̶: string | undefined = undefined // Deprecated - use `useDraco` instead
+useDraco: string | boolean = false
```

Comment:
Well done to think about deprecation messages! Do you think it's necessary to add a little hint in terms of a 
```md
:::admonition type="tip"
… 
:::
```
that you can provide a boolean and a string value to useDraco?

Reply:
Great, I'll create that tip rn, thanks for the hint :)